shared/nav.desktop.js

```diff
@@ -26,7 +26,8 @@ function Nav (props: Props) {
         />
       }
       <Box style={{...globalStyles.flexBoxColumn, flex: 1}}>
-        {props.children}
+        {props.routeStack.filterNot(r => r.tags.layerOnTop).last().component}
```

Comment:
not that this is a big perf thing but if we're just going to get the last item can't we just do a _.findLast() instead of filtering and then getting the last?
or maybe if we're actually dividing up the routestate we do _.partition?